package.json

```diff
@@ -76,6 +76,7 @@
     "lodash": "^4.17.20",
     "moment-timezone": "^0.5.33",
     "monaco-editor": "^0.25.2",
+    "notistack": "^3.0.1",
```

Comment:
How much file size does this add to our bundle?

Reply:
24.3 kB minified, 8.4kb minified + gzipped